srcpkgs/mesa-asahi/template

```diff
@@ -0,0 +1,192 @@
+# Template file for 'mesa-asahi'
+pkgname=mesa-asahi
+version=20241211
```

Comment:
why can't we just use mesa's version?

Reply:
Because they release by date. Or do you mean mesa_ver-date?

Reply:
mesaver+date could work

just having this done by date will make it annoying/hard to migrate over to regular mesa when the asahi stuff gets merged

Reply:
Makes sense. Done.